src/factors/Inertial/IMUDeltaFactor.jl

```diff
@@ -0,0 +1,467 @@
+using Manifolds
+using StaticArrays
+using Rotations
+using LinearAlgebra
+using DistributedFactorGraphs
+using Dates
+
+function TransformUtils.skew(v::SVector{3,T}) where T<:Real
+    return SMatrix{3,3,T}(
+            0,
+         v[3],
+        -v[2],
+        -v[3],  
+            0,  
+         v[1],
+         v[2],
+        -v[1],
+            0
+    )
+end
+
+struct IMUDeltaManifold <: AbstractManifold{ℝ} end
+
+# TODO maybe define the manifold like this
+# M = ProductManifold(TranslationGroup(3), TranslationGroup(3), SpecialOrthogonal(3), Euclidean(1))
+# M = ProductManifold(SpecialOrthogonal(3), TranslationGroup(3), TranslationGroup(3), Euclidean(1))
+# G = GroupManifold(M, MultiplicationOperation())
+
+#TODO 
+# also see SE₂(3) 
+
+const IMUDeltaGroup = GroupManifold{ℝ, IMUDeltaManifold, MultiplicationOperation}
+
+IMUDeltaGroup() = GroupManifold(IMUDeltaManifold(), MultiplicationOperation())
+
+Manifolds.manifold_dimension(::IMUDeltaManifold) = 9
+
+# Affine representation 
+# Δ = [ΔR Δv Δp;
+#      0   1 Δt;
+#      0   0  1] ⊂ \R^5x5
+
+#  ArrayPartition representation (TODO maybe swop order to [Δp; Δv; ΔR; Δt])
+# Δ = [ΔR; Δv; Δp; Δt] 
+
+# TODO is SemidirectProductGroup possible, Translation(3), Translation(3), SO(3), WhatAction?, Δt?
```

Comment:
Hm, maybe, but I think it will be easier to work without that. It has a few tricks that could be useful here but it's easier to just copy/adapt them.